### DISCUSSION TITLE
Name change

### DISCUSSION THREAD

**Discussion by MikkelDA** [MikkelDA/COBY#14] [Polls]
The abbreviation for Coarse Grained System Builder (CGSB) is not that good, primarily because of the "G", so i have decided to change it. I am a bit unsure about the new abbreviation so here is a poll to decide it.

Feel free to add more suggestions in the comments.

**Comment by Lp0lp** [comment]
COSY reminds me of NMR, COBY sounds cooler than CSB!

**Comment by MikkelDA** [comment]
Added poll options by request. Apparently that deletes previous answers so you need to vote again.

**Reply by MikkelDA** [reply]
Due to the poll, personal preference and in group talks, i have decided to rename CGSB to COBY from the next update and onwards. I will look into changing the GitHub name but it might require a completely new GitHub page.

**Comment by MikkelDA** [comment]
The name has now been changed to COBY and the program has been made pip installable using "pip install COBY".
